static/js/settings_ui.js

```diff
@@ -33,10 +33,13 @@ exports.do_settings_change = function (request_method, url, data, status_element
     var success_msg;
     var success_continuation;
     var error_continuation;
+    var removeAfter = 1000;
+
     if (opts !== undefined) {
         success_msg = opts.success_msg;
         success_continuation = opts.success_continuation;
         error_continuation = opts.error_continuation;
+        if(opts.sticky) removeAfter = null;
```

Comment:
Stylistically, we generally do space after `if` and curly braces for all conditionals, for readability